packages/instant/src/util/big_number.ts

```diff
@@ -0,0 +1,29 @@
+import { BigNumber } from '@0x/utils';
```

Comment:
Is there any way to achieve this using BigNumber's API rather than extending it? I'm a bit wary of us ending up in a situation where we don't know if something is a BigNumber / BigNumberInput, we get unexpected bugs, etc

Reply:
There isn't... We are only overriding `toString`... Maybe if a dependency depends on `toString` never returning an invalid number we could be in trouble.

How bout this. I'm going to change all references to `selectedAssetAmount` to be `BigNumberInput` and not override `toString` but add another method that returns the displayable string.